examples/qml_extension_plugin/tests/main.cpp

```diff
@@ -0,0 +1,8 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group
+// company <info@kdab.com> SPDX-FileContributor: Leon Matthes
```

Comment:
you need the clang format on/off otherwise it corrupts things, see other cpp files :-)

Reply:
Done